test/tools/unified-spec-runner/entities.ts

```diff
@@ -178,7 +178,12 @@ export class FailPointMap extends Map<string, Document> {
       // create a new client
       address = addressOrClient.toString();
       client = getClient(address);
-      await client.connect();
+      try {
+        await client.connect();
+      } catch (error) {
+        console.error(ejson`failed to connect enableFailPoint ${address}`);
```

Comment:
can we use regular templates here and below where the addresses are strings? just seems to add unnecessary cognitive load when trying to read the code where fancy templating isn't necessary